### ISSUE TITLE
Clarify meaning of number

### ISSUE THREAD

**Issue by Almad** [apiaryio/mson#50]
For `number` primitive type, specification says:

> Specifies any number.

I think it should be specified what `any` means. Integers only? Signed? Unsigned? Decimal points—fixed or floats?

If "any" truly means "all of the above", I think it would be useful to explicitly mention it in spec.

**Comment by honzajavorek** [comment]
Good point. Is `-1.7E+3` valid MSON number? This is how JSON Schema does it: http://spacetelescope.github.io/understanding-json-schema/reference/numeric.html

**Comment by zdne** [comment]
Related https://github.com/apiaryio/api-blueprint/issues/297
